tracer/test/snapshots/GraphQL2Tests.SubmitsTraces.verified.txt

```diff
@@ -0,0 +1,458 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /graphql,
+    Service: Samples.GraphQL,
+    Type: web,
+    Tags: {
+      component: aspnet_core,
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.status_code: 200,
+      http.url: http://localhost:00000/graphql,
```

Comment:
I'm surprised that the query string is not here...I can't tell if that's an issue or not

Reply:
So... this is something I saw ages ago and then forgot - we have discrepencies over when we include the querystring and when we don't 😬 We need to look into what we should do and what we are doing in general...

Reply:
Fwiw, `http.url` should _not_ include the query string by default, and users should be able to opt-in. We haven't added a setting for this in the .NET Tracer, though.

Reply:
Sounds good, we can tackle this at some other time. I created a JIRA ticket to track the missing setting